superset-frontend/src/components/Menu/SubMenu.tsx

```diff
@@ -53,19 +53,33 @@ const StyledHeader = styled.header`
     li.active > a,
     li.active > div,
     li > a:hover,
+    li > a:focus,
     li > div:hover {
-      background-color: ${({ theme }) => theme.colors.secondary.light4};
+      background: ${({ theme }) => theme.colors.secondary.light4};
       border-bottom: none;
       border-radius: 4px;
```

Comment:
```suggestion
      border-radius: ${({ theme }) => theme.borderRadius}px;
```
^ touching up someone else's work, but we don't get to break out this theme variable every day ;)